app/src/main/java/com/mapbox/vision/examples/activity/ar/ArNavigationActivity.kt

```diff
@@ -204,7 +206,8 @@ class ArNavigationActivity : AppCompatActivity(), RouteListener, ProgressChangeL
                             GeoCoordinate(
                                 latitude = geometryStep.latitude(),
                                 longitude = geometryStep.longitude()
-                            ), null
+                            ),
+                            null.mapToManeuverType()
```

Comment:
IMO direct `ManeuverType.None` looks nicer, but up to you

Reply:
Removed it at all as `ManeuverType.None` is constructor default value.